c7n/resources/health.py

```diff
@@ -51,7 +51,7 @@ class resource_type(object):
     def __init__(self, ctx, data):
         super(HealthEvents, self).__init__(ctx, data)
         self.queries = QueryFilter.parse(
-            self.data.get('query', [{'eventStatusCodes': 'open'}]))
+            self.data.get('query', [{'eventStatusCodes': 'open'},{'eventTypeCategories': 'scheduledChange'}]))
```

Comment:
re scheduled change filter, we actually want the inverse, since we want to defer resource specific (rds, ebs, ec2) health events to a filter on those respective resources.